lib/Mojo/mysql.pm

```diff
@@ -72,6 +72,10 @@ sub _dequeue {
   # you, silently, but only if certain env vars are set
   # hint: force-set mysql_auto_reconnect or whatever it's called to 0
   $dbh->{mysql_auto_reconnect} = 0;
+  # Maintain Commits with Mojo::mysql::Transaction
+  $dbh->{AutoCommit} = 1;
```

Comment:
Could you mention this in the documentation? Right now, the doc explicit mention AutoCommit: https://github.com/harry-bix/mojo-mysql/blob/no-sth-cache/lib/Mojo/mysql.pm#L194